block-core/src/transaction.rs

```diff
@@ -21,10 +26,21 @@ impl TransactionAction {
 
                 Address::from(M256::from(Keccak256::digest(rlp.out().as_slice()).as_slice()))
             },
+            &TransactionAction::Create2(salt, ref code) => {
+                let mut digest = Keccak256::new();
+                digest.input(&[0xff]);
+                digest.input(&caller);
+                digest.input(&salt);
+                digest.input(code);
```

Comment:
According to https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1014.md the hash of code should be used, not the actual value.

Reply:
https://github.com/ethereum/EIPs/pull/1014/files#r219843015

https://github.com/ethereum/EIPs/pull/1375